### PR TITLE
[SD-4010] On Updating the role of an user it is not properly reflecting privileges in the privileges tab.

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

File: scripts/superdesk-users/users.js [hunk 1]
```diff
@@ -1390,30 +1390,50 @@
...
+                            notify.error(gettext('User not found.'));
+                            console.log(error);
+                            return $q.reject(error);
+                        });
+                    }
 
-                    // the last user privileges that were saved on the server
-                    scope.origPrivileges = angular.copy(scope.user.privileges);
+                    function getPrivileges() {
+                        api('privileges').query().then(function(result) {
```
Comment (petrjasek) [inline scripts/superdesk-users/users.js]: seems like missing return? without this it won't be able to stop the promise chain

Comment (mdhaman) [inline scripts/superdesk-users/users.js]: yes

**Comment by mugurrus** [comment]
:+1: (a regression test would be good to have)

**Pull request merged**